src/pleasant-usart.h

```diff
@@ -113,6 +113,12 @@ void usart_read_bytes(uint8_t *bytes, size_t count, enum usart_error *error);
 void usart_write_string(char *characters);
 
 /*
+ * Writes all decimal digits of an integer to the USART. The most significant
+ * digit will be first. Leading 0's will be ignored.
+ */
+void usart_write_integer(uint8_t *integer);
```

Comment:
I'm not sure about the name. Maybe we should use the same convention as Arduino's Serial class, with `write` for raw values and `print` for human-readable values.

This could also be a little less specific about the kind of integers it supports, which seems less than ideal. Support for negative integers and integers of other sizes would be required.